spec/fixtures/mock_webserver_tpl.lua

```diff
@@ -28,9 +28,11 @@ http {
 
   server {
 # if protocol ~= 'https' then
-    listen ${http_port};
+    listen 127.0.0.1:${http_port};
+    listen [0000:0000:0000:0000:0000:0000:0000:0001]:${http_port};
```

Comment:
```suggestion
    listen [::1]:${http_port};
```

should be good enough. `::` can replace any consecutive `0`s in the IPv6 address.

There are a few places that can be changed like this.